test/e2e/scenarios/podselector.go

```diff
@@ -0,0 +1,45 @@
+package scenarios
+
+import (
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = PDescribe("[Dataplane] PodSelector based Network Policies for ingress between clusters", func() {
+	PContext("Allow communication between selected pods on different clusters in same project", func() {
+		PIt("Should allow communication between selected pods", func() {
+			By("creating listener pod 1 with label 1 in cluster 1 in namespace 1")
+			By("creating connecting pod 2 with label 2 in cluster 2 in namespace 2")
+			By("creating network policy in cluster 1 that allows communication to pod 1 from pod 2 in cluster 2")
```

Comment:
That shouldn't work I think, because they live in different namespaces they would not be selected by podselector, it's my understanding that you need namespaceSelector + podSelector combination for this, unless you put pod2 in namespace 1 too

Reply:
Yes, thats correct.

Reply:
I was thinking that "BeforeEach" of most of the tests we may need to create an "empty" network policy, because may be otherwise the tests will work even if we didn't created the policy. We will need to check.

For example, the connector pod will start attempting to connect the moment you create it, if there is no network policy preventing such connection, then it will succeed, connect, and exit.

We will figure out a way when we start implementing.